packages/integers/integers.0.1.0/opam

```diff
@@ -13,7 +13,7 @@ build:
            "--pinned" "%{pinned}%"]]
 depends: [
   "ocaml"
-  "ocamlbuild" {build}
+  "ocamlbuild" {build & >= "0.9.1"}
```

Comment:
This should be `& != "0.9.0"` as `ocamlbuild.0` is the only version available in OCaml < 4.03

Reply:
oh, sorry

Reply:
There is no easy button to revert one commit quickly and the whole PR could do with a few explanation of why they were failing in the first place. I'll split them all in several PRs coming right up.

Reply:
Fair point, thanks